.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: Storm WIKI
```

Comment:
I will remove this file because it is related with contribution policy and details https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks